t/t-locks.sh

```diff
@@ -72,6 +72,17 @@ begin_test "list a single lock (SSH; git-lfs-authenticate)"
   grep "f.dat" locks.log
   grep "Git LFS Tests" locks.log
   grep "lfs-ssh-echo.*git-lfs-authenticate /$reponame download" trace.log
+
+  GIT_TRACE=1 git -c lfs."$sshurl".sshprotocol=git-lfs-authenticate lfs locks --path "f.dat" 2>trace.log | tee locks.log
+  cat trace.log
```

Comment:
This `cat` and the ones in the subsequent test could probably be removed; I wondered if they were left over from the test construction and debugging process.

Reply:
I've removed them.